crates/test-macros/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "test-macros"
```

Comment:
Note that we already have: https://github.com/pragma-org/amaru/blob/main/crates/amaru-kernel/src/macros.rs

Reply:
ping @etorreborre

Reply:
@KtorZ as far as I understand it is not possible to mix `proc-macros` with `macro-export` functions, so it has to be a separate crate.